deploy/operator.yaml

```diff
@@ -57,7 +57,7 @@ spec:
             - name: OPERATOR_NAME
               value: "ibm-catalog-ui-operator"
             - name: OPERAND_CATALOG_UI_IMAGE
-              value: "quay.io/opencloudio/icp-catalog-ui@sha256:dbf31453dbb943a1ddd5a8d72fa89f3136232039f91e990148fdc5a060b75332"
```

Comment:
is this the next digest that will be pushed to quay?

Reply:
yes, https://travis.ibm.com/IBMPrivateCloud/catalog-ui/jobs/36351283#L288